pybossa/jobs.py

```diff
@@ -1014,9 +1014,10 @@ def get_weekly_admin_report_jobs():
     timeout = current_app.config.get('TIMEOUT')
     current_app.logger.info(u'Checking weekly report for admins, scheduled date: {}, today: {}'
                             .format(send_emails_date, today))
+    jobs = []
     if recipients and today == send_emails_date:
         info = dict(timestamp=datetime.now().isoformat(),
-            user_id="admin",
+            user_id=0,
```

Comment:
why changing user_id from "admin" to 0?

Reply:
because this report was kick off by user clicking "Export" on GIGwork UI, so it tracks the user's id, which is an integer. Now the report is generated automatically, there is no user id to fill, so I put 0 to distiguish auto export and user requested export action.
Other places are assuming user_id is integer, e.g. [here](https://github.com/bloomberg/pybossa/blob/50fc65c4ce02a82c1da680d5585f960cb2f39918/pybossa/exporter/csv_export.py#L63)